sdks/java/io/pom.xml

```diff
@@ -32,9 +32,36 @@
   <description>Beam SDK Java IO provides different connectivity components
   (sources and sinks) to consume and produce data from systems.</description>
 
+  <properties>
+    <hadoop.version>2.7.3</hadoop.version>
```

Comment:
Yes, It is exactly as you said, I will add the comment.
I moved from 2.7.0 to 2.7.3 because they use something like semver, so this version should include the latest bugfixes which I think is a good thing and they don't break API compatibility.
https://hadoop.apache.org/versioning.html
Since people should provide their runtime version, this should not be an issue because the change is only for a maintenance version. But if you disagree I can change it back to 2.7.0.